homeassistant/components/dnsip/sensor.py

```diff
@@ -52,6 +52,7 @@ class WanIpSensor(SensorEntity):
 
     _attr_icon = "mdi:web"
     _attr_has_entity_name = True
+    _retries = 2
```

Comment:
Since this is going to get immediately shadowed on the first update, and there doesn't seem to be any need to share the value between instances, wouldn't it be better to just declare in the initialiser?

Also the retry limit '2' is a magic number used in more than one place, so might be more maintainable to use a constant.